extension/src/popup/components/account/AccountHeader.tsx

```diff
@@ -0,0 +1,198 @@
+import React from "react";
+import { useSelector } from "react-redux";
+import styled from "styled-components";
+import { Link } from "react-router-dom";
+
+import { COLOR_PALETTE, ROUNDED_CORNERS } from "popup/constants/styles";
+import { ROUTES } from "popup/constants/routes";
+
+import { AccountListIdenticon } from "popup/components/identicons/AccountListIdenticon";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+
+import { Account } from "@shared/api/types";
+import CreateNewIcon from "popup/assets/create-new.svg";
+import ImportNewIcon from "popup/assets/import-new.svg";
+
+const AccountHeaderSection = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  height: 5rem;
+`;
+
+const AccountHeaderButtonEl = styled.div`
+  cursor: pointer;
+  display: flex;
+`;
+
+interface AccountHeaderOptionsProps {
+  dropdownCount: number;
+  isDropdownOpen: boolean;
+}
+
+const AccountHeaderOptionsEl = styled.ul`
+  background: ${COLOR_PALETTE.white};
+  border-radius: 0.3125rem;
+  box-shadow: 0 0.125rem 0.5rem rgba(0, 0, 0, 0.06);
+  height: auto;
+  max-height: ${({
+    dropdownCount,
+    isDropdownOpen,
+  }: AccountHeaderOptionsProps) =>
+    isDropdownOpen ? `${dropdownCount * 6.5}rem` : "0"};
```

Comment:
this might be a bit of a challenge to do with pure CSS. We can chat about the best approach here!

Reply:
ya  another reason I wanted to do in another PR haha, not sure how to do this in scss atm (or if it's even needed with the style changes still needed)